modules/microshift-config-parameters-table.adoc

```diff
@@ -65,27 +65,151 @@ The following table explains {microshift-short} configuration YAML parameters an
+
+This value is applied globally as a default for all routes, but can be overridden per-route by the route annotation `router.openshift.io/haproxy.health.check.interval`.
+
+Requires an unsigned duration string of decimal numbers, each with an optional fraction and unit suffix, such as `300ms`, `1.5h` or `2h45m`. Valid time units are `ns`, `us` (or `µs` U+00B5 or `μs` U+03BC), `ms`, `s`, `m`, `h`.
+
+Setting this parameter value to less than `5s` can cause excess traffic due to too frequent TCP health checks and accompanying SYN packet storms.
+
+Setting this parameter value too high can result in increased latency because of backend servers that are no longer available, but have not yet been detected as such.
+
+An empty or `0` value means "no opinion" and the `IngressController` chooses a default which is subject to change over time.
+
+|`ingress.tuningOptions.maxConnections`
+|`integer`, valid values are: `empty`, `0`, `-1`, and the range `2000-2000000`
+|Default value is `0`. defines the maximum number of simultaneous connections that can be established per `HAProxy` process. Increasing this value allows each ingress controller pod to handle more connections at the cost of additional system resources being consumed.
+
+If this field is empty or `0`, the `IngressController` uses the default value of `50000`, but the default is subject to change in future releases.
+
+If the value is `-1` then `HAProxy` dynamically computes a maximum value based on the available resources set with `ulimit` values in the running container. Selecting `-1`, which means `auto`, results in a large value being computed, and therefore each `HAProxy` process incurs significant memory usage compared with the current default of `50000`.
+
+Setting a value that is greater than the current operating system limit prevents the `HAProxy` process from starting.
+//* If you choose a discrete value, for example, `750000`, and the router pod is migrated to a new node, there's no guarantee that that new node has identical ulimits configured. In such a scenario the pod would fail to start. If you have nodes with different ulimits configured (e.g., different\tuned profiles) and you choose a discrete value then the guidance is to use -1 and let the value be computed dynamically at runtime.
+//Does this apply to MicroShift which is single-node?
+
+You can monitor memory usage for router containers with the following metric: `container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingress`}`. You can monitor memory usage of individual `HAProxy`processes in router containers with the following metric: `container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingress`}/container_processes{container=`router`,namespace=`openshift-ingress`}`.
```

Comment:
These aren't rendering properly, and would probably need a confusing-level of escapes. Maybe switch these ones to code blocks instead of in-line code/backticks?
```suggestion
You can monitor memory usage for router containers with the following metric:

[source,terminal]
----
container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingress`}`
----

You can monitor memory usage of individual `HAProxy`processes in router containers with the following metric:

[source,terminal]
----
container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingress`}/container_processes{container=`router`,namespace=`openshift-ingress`}
----
```

Note this would also require `a` set at the start of the cell's `|` ([line 138](https://github.com/openshift/openshift-docs/pull/84548/files#diff-5fdd35dcb5f1bf5c78425d5ba6f3bedbf137bef6b93963d86e944b78a7a0e7cbR178)). 

FWIW, for both the bullets suggestion and this code block suggestion, you can alternatively set `a` at the [column level instead](https://docs.asciidoctor.org/asciidoc/latest/tables/format-column-content/) and have it auto-apply to all of its cells, instead of having to apply per cell. I think for this table that would be:

```
[cols="3a","20%,10%,70%",options="header"]
```
^ adding the `a`